### PR TITLE
Updated README file to correct the code to bind routes to the http server

### PR DESCRIPTION
**Pull request by er-rishi**
- Binidng the routes with record metrics to http serve is not having the correct code so i have updated the same.

### CODE REVIEW

**Comment by rkhandelwal-rms** [comment]
@RustedBones Please review the PR and let me know if any other changes required.

File: README.md [hunk 1]
```diff
@@ -80,7 +81,8 @@ val registry: HttpMetricsRegistry = ... // concrete registry implementation
 
 val route: Route = ... // your route
 
-Http().newMeteredServerAt("localhost", 8080, registry).bindFlow(route)
+val routesWithMetrics = HttpMetricsRoute(routes).recordMetrics(registry)
+Http().bindAndHandle(routesWithMetrics, "localhost", 8080)
```
Comment (RustedBones) [inline README.md:85]: This is the deprecated `akka-http` API. if you are still using akka 10.1, Use version [`v1.1.1`](https://github.com/RustedBones/akka-http-metrics/tree/v1.1.1) of the library

Comment (er-rishi) [reply README.md:85]: In v1.1.1 version also, I was not able to get the function newMeteredServerAt()

Comment (RustedBones) [reply README.md:85]: - In `v1.1.1` use 
```scala
Http().bindAndHandle(route.recordMetrics(registry), "localhost", 8080)
```
If you navigate to the [README.md for this release](https://github.com/RustedBones/akka-http-metrics/tree/v1.1.1) the doc is correct

- In `v1.2.0` use
```scala
Http().newMeteredServerAt("localhost", 8080, registry).bindFlow(route)
```
The README.md on the project page is from the `master` branch. It documents the latest (and potentially unreleased) version of the lib

Comment (RustedBones) [reply README.md:85]: One side note, for version `v.1.2.0` to get access to `newMeteredServerAt` you must not forget to import implicits like 
```scala
import fr.davit.akka.http.metrics.core.HttpMetrics._
```

**Comment by RustedBones** [comment]
@er-rishi any luck making it work ?

**Comment by er-rishi** [comment]
> @er-rishi any luck making it work ?

It is working for me. I used v1.1.1, but still in doubt whether this newMeteredServerAt() is correct or not

**Comment by RustedBones** [comment]
Great to hear.

As mentioned in the comments, this is part of the new API, starting from version `1.2.0`. if you want to consult documentation from previous release, check the corresponding tag.

I'll close this PR

**Pull request closed without merging**